pkg/terraform/util.go

```diff
@@ -87,3 +88,19 @@ func CreateTerraformBackendFile(assetDir, data string) error {
 
 	return nil
 }
+
+// Ensures a directory named path exists. If the directory doesn't exist, the function creates it
+// including any missing parents along the path.
+func ensureDirExists(path string) error {
+	pathExists, err := util.PathExists(path)
+	if err != nil {
+		return errors.Wrapf(err, "failed to stat path %q: %v", path, err)
+	}
+	if pathExists {
+		return nil
+	}
+	if err := os.MkdirAll(path, 0755); err != nil {
```

Comment:
Isn't `os.MkdirAll` idempotent? We can just call that and then we don't need helper function at all.

Reply:
Good point, I'm not sure why we've done it this way in the first place. Probably an oversight. Fixing.